source/getting-started/quickstart.md

```diff
@@ -0,0 +1,262 @@
+This guide will teach you how to build a simple app using Ember from
+scratch.
+
+We'll cover these steps:
+
+1. Installing Ember
+2. Creating a new application
+3. Defining a route
+4. Writing a UI component
+5. Building your app to be deployed to production
+
+## Install Ember
+
+You can install Ember with a single command using npm, the Node.js package
+manager. Type this into your terminal:
+
+```sh
+npm i ember-cli -g
```

Comment:
We've tended to prefer full command names over the short versions, for clarity to newer people.